handbook/data-stores/mongodb.md

```diff
@@ -0,0 +1,133 @@
+# MongoDB
+
+The majority of SystemLink services use [MongoDB](https://www.mongodb.com) as the primary database. Default installations of SystemLink include MongoDB Community Edition, which launches when the SystemLink server boots. This is referred to as a [*single node*](#single-node-deployments) deployment.
+
+You can also host MongoDB on a separate standalone server, a [replica set](https://docs.mongodb.com/manual/replication/) of three or more servers, or use MongoDB's platform as a service offering, [MongoDB Atlas](https://www.mongodb.com/cloud/atlas). This is referred to as a [*multi node*](#multi-node-deployments) deployment.
+
+## Assumptions and Prerequisites
+
+Before you configure a remote MongoDB instance to use with SystemLink, ensure you have the following:
+
+- A server running SystemLink 2021 R1 or greater. Refer to [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server.
+- A standalone server running MongoDB, multiple servers hosting a MongoDB replica set, or a MongoDB Atlas Account
+- A user with the `readWriteAnyDatabase` or similar role. Refer to [Role-Based Access Control](https://docs.mongodb.com/manual/core/authorization/) for details. You reference this user in the MongoDB connection fields or connection string in **NI SystemLink Server Configuration**.
+
+!!!important
+    Due to a bug in a third-party MongoDB driver, SystemLink cannot connect to MongoDB instances where the MongoDB username either the `-` or the `_` character.
+
+## Single Node vs Multi Node MongoDB Deployments
+
+The following table summarizes when to use the various supported configurations of MongoDB with SystemLink.
+
+| Diagram | SystemLink + MongoDB Configuration | When to use |
+| | ----------- | ----------- |
+|<img src="../../img/single-box.png" width="1500px"/>| Single node with defaults     | You have one server hosting both MongoDB and SystemLink. Use this during evaluation, when working with less than 10 managed nodes, and in deployments that don't need the Asset Manager and Test Insights modules.
+|<img src="../../img/single-box.png" width="1500px"/>| Single node with increased index cache   | You have one server hosting both MongoDB and SystemLink. Use this during evaluation, in moderate deployments of less than 25 managed nodes, and for moderate usage of the Test Insights module producing less than 100 test steps and results each day.  |
```

Comment:
This guidance is always difficult. We originally set index sizing guidance based on 1GB per 10,000 test steps. If we give a limit here we should target a single object type (results or steps). Does the per step estimate help define the moderate use-case?